### DISCUSSION TITLE
PiHole Block Lists

### DISCUSSION THREAD

**Discussion by smugdruggler** [StevenBlack/hosts#1624] [Q&A]
Hi, I'm not sure if this is the right place to raise this and if not I apologise.

It would seem that the block list maintained by Steven Black which I hope is you has a couple of entries that no longer work and I wonder if they can be removed?

https://hosts-file.net/ad_servers.txt
https://mirror1.malwaredomains.com/files/justdomains

Thanks.

**Comment by StevenBlack** [comment]
Hi @smugdruggler these entries in Pi-Hole have nothing to do with us.
